adapters/balance.js

```diff
@@ -38,6 +38,14 @@ module.exports = function (mu, transports) {
 
 
 
+  function setMu (muInstance) {
+    transports.forEach(function (transport) {
+      transport.setMu(muInstance)
+    })
+  }
```

Comment:
Why this? I think we can avoid setMu for just a transport.mu = mu. The setter can be overloaded if needed.

The same holds true for the id.